Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "async-compression"
-version = "0.3.15"
+version = "0.4.0"
```

Comment:
The script I mentioned in #200 can now keep this in sync with the changelog.

Reply:
Ok, I'll drop that commit and rebase over #200 then.